### PR TITLE
#159917722 Implementation of LFA feedback

### PR DESCRIPTION
**Pull request by PriscillaSam**
#### What does this PR do?
Implement LFA feedback on hosted UI templates.

#### Description of Task to be completed?
* Change home page banner text color
* Add top margin to login form
* Add a modal to post question on question page
* Change read more text color on answer cards

#### How should this be manually tested?
* Open a terminal and run `git clone https://github.com/PriscillaSam/Stackoverflow-Lite.git`.
* run `git checkout ft-implement-feedback-159917722`.
* open up the project in a file explorer and open home.html on a browser.

#### Any background context you want to provide?
None.

#### What are the relevant pivotal tracker stories?
#159917722

### CODE REVIEW

File: UI/css/main.css [hunk 1]
```diff
@@ -90,17 +92,21 @@ padding-bottom: 20px;
...
+     padding: 50px;
 
+}
 .box{
     border-radius: 5px;
     background-color: white;
     padding: 5px 10px;
     margin-bottom: 10px;
+    box-shadow: 0 0 8px 2px rgba(0,0,0,0.1);
+
```
Comment (Billmike) [inline UI/css/main.css]: No need for this extra line.

File: UI/css/main.css [hunk 2]
```diff
@@ -90,17 +92,21 @@ padding-bottom: 20px;
...
+    background-image: url('../img/13.jpg');
     background-size: cover;
-    height: 350px;   
-    padding: 20px 0 20px 50px; 
 }
+.overlay{
+    background-color: rgba(0, 0, 0, 0.7);
+     padding: 50px;
 
+}
```
Comment (Billmike) [inline UI/css/main.css]: ensure that there is a space after a closing curly brace and the next class/element/id style definition. Something like this:

```css
.overlay{
    background-color: rgba(0, 0, 0, 0.7);
     padding: 50px;
 }

.overlay{
    background-color: rgba(0, 0, 0, 0.7);
     padding: 50px;
 }
```

File: UI/css/main.css [hunk 3]
```diff
@@ -308,8 +315,10 @@ margin-top: 10px;
   
 }
 @media(max-width:640px){
-    .banner{
-        height: 250px;
+
```
Comment (Billmike) [inline UI/css/main.css]: No need for this space.

File: UI/css/style.css [hunk 4]
```diff
@@ -106,8 +112,8 @@ h2,
     font-weight: 1;
 }
 .bg-light{
-    background-color: #E6E6FA;
-
+    background-color: #EEE8AA;
+    /* background-color: #DcDcDc; */
```
Comment (Billmike) [inline UI/css/style.css]: You can lose this code if it's not needed anymore.

File: UI/css/style.css [hunk 5]
```diff
@@ -180,4 +187,7 @@ textarea{
     width: 50%;
     bottom: 100px;
     left: 25%;
+    max-height: 100%;
+    overflow: auto;
+
 }
```
Comment (Billmike) [inline UI/css/style.css]: Add a new line here.

**Review by Billmike — CHANGES_REQUESTED** [review]

**Comment by Billmike** [comment]
Great work with the improvements you have made to the UI. It's looking great! Just a quick one though: Go through the images and you would see how your application looks currently on large screens

screen shot 2018-08-21 at 6 48 47 pm

screen shot 2018-08-21 at 6 48 56 pm

screen shot 2018-08-21 at 6 49 00 pm

screen shot 2018-08-21 at 6 49 11 pm

You might want to write media query to take care of that. You can look-up this great resource for defining screen sizes [here](https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints/)

File: UI/css/main.css (hunk 2 above)
Comment (PriscillaSam) [reply UI/css/main.css]: Noted. Thanks

**Pull request merged**